app/views/pages/dashboard.html.erb

```diff
@@ -55,9 +55,33 @@
         <% end %>
         </div>
         <div class="tab-pane fade" id="previous-jobs" role="tabpanel" aria-labelledby="previous-jobs-tab">
-          <h2>You don't have any current or previous jobs.</h2>
-          <p>When a company accepts your application, the job will appear here.</p>
-          <%= link_to "View all jobs", search_path %>
+          <% if @developer_jobs.present? %>
+            <% @developer_jobs.each do |job| %>
+              <%= link_to job_path(job) do %>
```

Comment:
could you add styling to remove the blue underlines on hover over this link? 
i.e. `text-decoration: none` on the approprate class